buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -574,6 +574,10 @@ class BeamModulePlugin implements Plugin<Project> {
       return project.containerArchitectures().collect { arch -> "linux/" + arch }
     }
 
+    project.ext.useBuildx = {
+      return project.containerArchitectures() != [project.nativeArchitecture()]
```

Comment:
Really, we just need to use buildx any time we use more than one architecture. So we should actually probably just gate this on the length of `containerArchitectures`

Reply:
buildx will still be used to build amd64 container on M1 Mac (in which I am actually using it)

Reply:
Oh, good call - nevermind I got my wires crossed.